### PR TITLE
tests: add new test for override example

### PR DESCRIPTION
**Pull request by jor2**
### Description

Test for override example in landing zone.

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

##### Release notes content

Test the override example in other_tests.go.

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Review by toddgiguere — CHANGES_REQUESTED** [review]
I think this test is a duplicate of this one that already exists?
https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/blob/main/tests/pr_test.go#L308

Also this particular test is not really doing anything, I think what it was meant to do is use the `config` output from first  apply as the override for the second apply? But setting that override doesn't take place in this test.

File: tests/other_test.go [hunk 1]
```diff
@@ -48,3 +52,29 @@ func TestRunVSIPatternWithHPCS(t *testing.T) {
...
+
+	if assert.Nil(t, err, "This should not have errored") &&
+		assert.NotNil(t, output, "Expected some output") &&
+		assert.NotNil(t, options.LastTestTerraformOutputs, "Expected some Terraform outputs") {
+
+		// TERRATEST uses its own internal logger.
+		// The "show" command will produce a very large JSON to stdout which is printed by the logger.
+		// We are temporarily turning the terratest logger OFF (discard) while running "show" to prevent large JSON stdout.
+		options.TerraformOptions.Logger = logger.Discard
+		planStruct, planErr := terraform.InitAndPlanAndShowWithStructE(options.Testing, options.TerraformOptions)
```
Comment (toddgiguere) [inline tests/other_test.go:71]: RunTestConsistency() will already run an Apply and then follow it up with a "InitAndPlan" to verify idempotent. I'm not sure what a second "InitAndPlan" will do here?

**Review by toddgiguere — CHANGES_REQUESTED** [review]

**Comment by jor2** [comment]
> I think this test is a duplicate of this one that already exists? https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/blob/main/tests/pr_test.go#L308
> 
> Also this particular test is not really doing anything, I think what it was meant to do is use the `config` output from first apply as the override for the second apply? But setting that override doesn't take place in this test.

That test runs the `quickStartPatternTerraformDir` and does that weird overriding using previous tests output. Should I replace that test with this one?

**Review by toddgiguere — APPROVED** [review]
I see the difference now, the other override test I was referring to is more of a "config output" test, and using override. This test is just testing our new example, so I do not think there is an overlap.

**Pull request merged**